manim/mobject/geometry.py

```diff
@@ -743,6 +742,54 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """
+    The ArcPolygon is what it says, a polygon, but made from arcs.
+    More versatile than the standard Polygon.
+
+    Parameters
+    ----------
+    *arcs : Arc or ArcBetweenPoints
+    
+    Example
+    -------
+    ArcPolygon(arc0,arc1,arc2,arcN,**kwargs)
+
+    
+    For proper appearance the arcs should seamlessly connect:
+    [a,b][b,c][c,a]
+    If they don't, the gaps will be filled in with straight lines.
+
+    ArcPolygon also doubles as a group for the input arcs.
+    That means these arcs can be drawn and seen separately.
+    If only the ArcPolygon itself is supposed to be visible,
+    make the arcs invisible (for example with "stroke_width": 0).
```

Comment:
I've reworked the explanation into this:
```
    ArcPolygon doubles as a VGroup for the input arcs, so it stores
    the passed arcs like a VGroup would, but also additionally uses the
    arcs to generate new points for itself.
    This is so that the ArcPolygon can be directly manipulated while
    assuring that the stored arcs, accessible via ArcPolygon.arcs, still
    return correct values.

    Because arcs are stored like this, if only the generated ArcPolygon
    itself is supposed to be visible, the passed arcs have to be made
    invisible (for example with "stroke_width": 0).
```
Is that clear to you or do you see more room for improvement?